cmd/caa-log-checker/main.go

```diff
@@ -79,19 +82,39 @@ func checkIssuances(scanner *bufio.Scanner, checkedMap map[string][]time.Time, s
 		}
 
 		var badNames []string
+		var minTimeError time.Duration = math.MaxInt64
+
 		for _, name := range ie.Names {
 			nameOk := false
+
 			for _, t := range checkedMap[name] {
-				if t.Before(ie.issuanceTime) && t.After(ie.issuanceTime.Add(-8*time.Hour)) {
+				validStart := ie.issuanceTime.Add(-8 * time.Hour)
+				validEnd := ie.issuanceTime
+				if t.After(validStart) && t.Before(validEnd.Add(timeTolerance)) {
 					nameOk = true
+				} else {
+					// If the check didn't pass, calculate how much tolerance we'd need for it to pass, to make it easier to diagnose log timestamp desync.
```

Comment:
Could you wrap this comment line?

Reply:
I naively assumed gofmt would deal with that for me. Oh well :)